repo-infra/verify/boilerplate/BUILD

```diff
@@ -0,0 +1,3 @@
+package(default_visibility = ["//visibility:public"])
+
+exports_files(glob(["*.txt"]))
```

Comment:
BUILD files are indeed for Bazel. I can't speak to whether we _want_ Bazel (and therefore whether we want to bring these files in).

There was a related conversation elsewhere - can we avoid checking in code-generated files? With Bazel think we should be able to  because Bazel has good support for things like that. With makefiles I don't see how to (easily) do it, particularly if the tools that generate the files come from different repos etc.